feature-group-definitions/input-show-picker.yml

```diff
@@ -0,0 +1,14 @@
+name: input.showPicker()
```

Comment:
Since it's fairly common to have methods by the same name that do the same (ish) thing spread out across DOM interfaces, I think we need a convention that doesn't require us to spell out interface name(s) in all cases. Let's consider these cases:

- `showPicker()` on `<input>` and `<select>` elements
- `fastSeek()` on `<audio>` and `<video>` elements (via `HTMLMediaElement`)
- `requestVideoFrameCallback()` on `<video>` elements (via `HTMLVideoElement`)
- `checkValidity()` on `<form>`, `<input>`, `<select>`, `<textarea>`, `<output>`, `<fieldset>`, and `<button>` (via many interfaces)

I think we should treat the element as disambiguation, so the method name should come first. So my preference would be "`showPicker()` for `<input>`" as the shortest plausible name I can come up with.

The other examples don't need disambiguation.